configs/platforms/aix-6.1-ppc.rb

```diff
@@ -3,18 +3,45 @@
 
   plat.make "gmake"
   plat.tar "/opt/freeware/bin/tar"
-  plat.rpmbuild "/usr/bin/rpm"
+  plat.rpmbuild "/usr/bin/rpmbuild"
   plat.patch "/opt/freeware/bin/patch"
+  plat.mktemp "/opt/freeware/bin/mktemp --directory --tmpdir=/var/tmp"
+  os_version = 6.1
+  plat.environment "CC", "/opt/pl-build-tools/bin/gcc"
 
-  # Basic vanagon operations require mktemp, rsync, coreutils, make, tar and sed so leave this in there
-  plat.provision_with "rpm -Uvh --replacepkgs http://osmirror.delivery.puppetlabs.net/AIX_MIRROR/mktemp-1.7-1.aix5.1.ppc.rpm http://osmirror.delivery.puppetlabs.net/AIX_MIRROR/rsync-3.0.6-1.aix5.3.ppc.rpm http://osmirror.delivery.puppetlabs.net/AIX_MIRROR/coreutils-5.2.1-2.aix5.1.ppc.rpm http://osmirror.delivery.puppetlabs.net/AIX_MIRROR/sed-4.1.1-1.aix5.1.ppc.rpm http://osmirror.delivery.puppetlabs.net/AIX_MIRROR/make-3.80-1.aix5.1.ppc.rpm http://osmirror.delivery.puppetlabs.net/AIX_MIRROR/tar-1.22-1.aix6.1.ppc.rpm"
+  # For pl-build-tools, we can't rely on yum, and rpm can't download over https on AIX, so curl packages before installing them
```

Comment:
A few of these comments in the platform files need updating, but otherwise LGTM!